model/src/main/kotlin/Project.kt

```diff
@@ -180,6 +180,11 @@ data class Project(
      */
     override fun compareTo(other: Project) = id.compareTo(other.id)
 
+    /**
+     * Return whether the package identified by [id] is contained as a (transitive) dependency in this project.
+     */
+    operator fun contains(id: Identifier) = scopes.any { id in it }
```

Comment:
So how about calling this `dependsOn` instead?

Reply:
I chose this for consistency with `Scope` where the function is also called `contains`. The reasoning was that it allows to write `id in scope`.